### PR TITLE
skipping ci on doc changes

### PR DESCRIPTION
**Pull request by JaydipGabani**
Adding `skip_if_only_changed: "^docs/|\.(md|adoc)$|^(README|LICENSE)$"` to skip the ci on doc change

### CODE REVIEW

File: ci-operator/jobs/konveyor/mig-operator/konveyor-mig-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ presubmits:
     cluster: build01
     context: ci/prow/ci-index
     decorate: true
+    skip_if_only_changed: "^docs/|\.(md|adoc)$|^(README|LICENSE)$"
```
Comment (shawn-hurley) [inline ci-operator/jobs/konveyor/mig-operator/konveyor-mig-operator-master-presubmits.yaml:10]: doesn't this say only run when docs changed?

Comment (JaydipGabani) [reply ci-operator/jobs/konveyor/mig-operator/konveyor-mig-operator-master-presubmits.yaml:10]: I dont think so, The document says the following.
```
skip_if_only_changed skips the job if all paths match. For example, you may wish to skip a compilation job for pull requests that only change documentation files:

presubmits:
  org/repo:
  - name: compile-job
    always_run: false
    skip_if_only_changed: "^docs/|\.(md|adoc)$|^(README|LICENSE)$"
    
    
Both of the above examples would trigger on a pull request containing foo/bar.c and SECURITY.md, but not one containing only SECURITY.md.

```
So a PR raised only with doc change should not trigger anything in this case.

**Review by jmontleon — APPROVED** [review]

File: ci-operator/jobs/konveyor/mig-operator/konveyor-mig-operator-master-presubmits.yaml (hunk 1 above)
Comment (shawn-hurley) [reply ci-operator/jobs/konveyor/mig-operator/konveyor-mig-operator-master-presubmits.yaml:10]: I wonder if using a single-quoted string will work better?

**Pull request merged**